pkg/operator/ceph/csi/spec.go

```diff
@@ -268,6 +269,21 @@ func startDrivers(clientset kubernetes.Interface, rookclientset rookclient.Inter
 		return errors.Wrap(err, "failed to load CSI_PROVISIONER_PRIORITY_CLASSNAME setting")
 	}
 
+	// OMAP generator will be enabled by default
+	// If AllowUnsupported is set to false and if CSI version is less than
+	// <3.2.0 disable OMAP generator sidecar
+	if v.SupportOMAPController() {
```

Comment:
Suggest changing this to `SupportsOMAPController` or `OMAPControllerIsSupported` to be more grammatically correct.

Reply:
done